src/main/java/org.fundacionjala.katapractice/HighAndLow.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.katapractice;
+
+import java.util.Arrays;
+
+/**
+ * Class to validate high and low number.
+ */
+public final class HighAndLow {
+
+    /**
+     * Private constructor.
+     */
+    private HighAndLow() {
+
+    }
+
+    /**
+     * Method to add get the high and low number.
+     *
+     * @param values the String that contain the values.
+     * @return the result of the calculation that is the high and low number.
+     */
+    public static String getHighLow(final String values) {
```

Comment:
I have tried  to do this change but there is a problem when I compile, the following exception is displayed "java.lang.IllegalStateException: stream has already been operated upon or closed" 
I research about the problem and, I read that a stream is possible to use only one time.